### PR TITLE
Update README.md with conan install 1.*

### PR DESCRIPTION
**Pull request by MathiasDanzeisen**
Changed the installed conan version to same as in docker build conatiner. To avoid the issue that the install conan now defaults to conan 2.* which is to completly compatable with 1.*

### CODE REVIEW

File: someip2val/README.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ SOME/IP feeder is [COVESA / vsomeip](https://github.com/COVESA/vsomeip/) applica
...
     ``` bash
     sudo apt-get install -y python3 python3-pip
-    pip3 install conan
+    pip3 install "conan==1.55"
     ```
     **NOTE:** Sometimes latest conan recipe revisions are broken, but the local build succeeds using cached older revision. If build fails on CI local conan cache could be cleared to reproduce the error. Also latest recipes may require newer conan version.
     ``` bash
     rm -rf ~/.conan/data
-    pip3 install -U conan
+    pip3 install -U "conan==1.55"
```
Comment (int0x27) [inline someip2val/README.md:58]: -U is redundant with fixed version here, better add to Note that  conan must be < 2.0, due to issues with aarch64 builds.
Another option would be:
```
pip3 install -U "conan<2.0"
```
But there is a hardcoded recipe in `conanfile.txt` that may need attention after updating conan version

**Review by int0x27 — CHANGES_REQUESTED** [review]

File: someip2val/README.md (hunk 1 above)
Comment (MathiasDanzeisen) [reply someip2val/README.md:58]: Ok, how about this?
```suggestion
    pip3 install "conan==1.*"
```
the reason for the PR is that I got conan 2 installen then the build script failed (I think "conan info")

Comment (int0x27) [reply someip2val/README.md:58]: Looks fine

**Review by int0x27 — APPROVED** [review]
looks good

**Pull request merged**